support-frontend/assets/pages/weekly-subscription-checkout/helpers/deliveryDays.ts

```diff
@@ -59,14 +64,54 @@ function getTierThreeDeliveryDate(today?: number) {
 	return result;
 }
 
-function getGuardianAdLiteDate(today?: number) {
-	const firstValidDate = addDays(new Date(today ?? new Date()), 15);
-	return firstValidDate;
-}
+const productDeliveryDate = (
+	productKey: ActiveProductKey,
+	paperProductOptions?: ActivePaperProductOptions,
+): Date | undefined => {
+	console.log(paperProductOptions);
```

Comment:
Should this `console.log` be removed before merging?

Reply:
Yes, should of spotted that, removed.